test/models/driver_test.rb

```diff
@@ -1,80 +1,149 @@
 require "test_helper"
 
 describe Driver do
-  let (:new_driver) {
-    Driver.new(name: "Kari", vin: "123", active: true,
-               car_make: "Cherry", car_model: "DR5")
-  }
-  it "can be instantiated" do
-    # Assert
-    expect(new_driver.valid?).must_equal true
-  end
-
-  it "will have the required fields" do
-    # Arrange
-    new_driver.save
-    driver = Driver.first
-    [:name, :vin, :active, :car_make, :car_model].each do |field|
-
+  let (:new_driver) { Driver.new(name: "Kari", vin: "123", available: true, car_make: "Cherry", car_model: "DR5") }
+  let (:unavailable_driver) { Driver.new(name: "Ben", vin: "456", available: false, car_make: "Honda", car_model: "Fit") }
+  let (:new_passenger) { Passenger.create(name: "Kari", phone_num: "111-111-1211") }
+  
+  describe "initialize" do
+    it "can be instantiated" do
       # Assert
-      expect(driver).must_respond_to field
+      expect(new_driver.valid?).must_equal true
+    end
+    
+    it "will have the required fields" do
+      # Arrange
+      new_driver.save
+      driver = Driver.first
+      [:name, :vin, :available, :car_make, :car_model].each do |field|
+        
+        # Assert
+        expect(driver).must_respond_to field
+      end
     end
   end
-
+  
   describe "relationships" do
     it "can have many trips" do
       # Arrange
       new_driver.save
-      driver = Driver.first
-
+      new_passenger.save
+      
+      trip_1 = Trip.create(driver_id: new_driver.id, passenger_id: new_passenger.id, date: Date.today, rating: 5, cost: 1234)
+      trip_2 = Trip.create(driver_id: new_driver.id, passenger_id: new_passenger.id, date: Date.today, rating: 3, cost: 6334)
+      
       # Assert
-      expect(driver.trips.count).must_be :>=, 0
-      driver.trips.each do |trip|
+      expect(new_driver.trips.count).must_equal 2
+      
+      new_driver.trips.each do |trip|
         expect(trip).must_be_instance_of Trip
       end
     end
   end
-
+  
   describe "validations" do
     it "must have a name" do
       # Arrange
       new_driver.name = nil
-
+      new_driver.save
+      
       # Assert
       expect(new_driver.valid?).must_equal false
       expect(new_driver.errors.messages).must_include :name
       expect(new_driver.errors.messages[:name]).must_equal ["can't be blank"]
```

Comment:
This test fails because you changed your message to "Name can't be blank." Consider using something like:
```
expect(new_driver.errors.messages[:name].downcase).must_include "blank"
```